packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -34,43 +51,108 @@ describe("<DateRangeInput>", () => {
 
-        // deselect days
-        getDayElement(22).simulate("click");
-        getDayElement(24).simulate("click");
+            const { root, getDayElement } = wrap(<DateRangeInput defaultValue={defaultValue} />);
+            root.setState({ isOpen: true });
 
-        // TODO: Make these checks more rigorous once controlled mode is supported,
-        // when we'll be able to enforce which initial month is shown.
-        expect(onChange.callCount).to.deep.equal(4);
-    });
+            // verify the default value
+            assertInputTextsEqual(root, START_STR, "");
 
-    it("shows empty fields when no date range is selected", () => {
-        const onChange = sinon.spy();
-        const { root } = wrap(<DateRangeInput onChange={onChange} />);
+            getDayElement(END_DAY).simulate("click");
+            assertInputTextsEqual(root, START_STR, END_STR);
 
-        expect(getStartInputText(root)).to.be.empty;
-        expect(getEndInputText(root)).to.be.empty;
-    });
+            getDayElement(START_DAY).simulate("click");
+            assertInputTextsEqual(root, "", END_STR);
+
+            getDayElement(END_DAY).simulate("click");
+            assertInputTextsEqual(root, "", "");
 
-    it("shows formatted dates in fields when date range is selected", () => {
-        const onChange = sinon.spy();
-        const { root, getDayElement } = wrap(<DateRangeInput onChange={onChange} />);
-        root.setState({ isOpen: true });
+            getDayElement(START_DAY).simulate("click");
+            assertInputTextsEqual(root, START_STR, "");
+        });
 
-        getDayElement(22).simulate("click");
-        getDayElement(24).simulate("click");
+        it("Clearing the date range clears the inputs, and invokes onChange with [null, null]", () => {
+            const onChange = sinon.spy();
+            const defaultValue = [START_DATE, null] as DateRange;
 
-        // TODO: Make these checks more rigorous once controlled mode is supported.
-        expect(getStartInputText(root)).to.be.not.empty;
-        expect(getEndInputText(root)).to.be.not.empty;
+            const { root, getDayElement } = wrap(<DateRangeInput defaultValue={defaultValue} onChange={onChange} />);
+            root.setState({ isOpen: true });
+
+            getDayElement(START_DAY).simulate("click");
+
+            expect(getStartInputText(root)).to.be.empty;
+            expect(getEndInputText(root)).to.be.empty;
+            expect(onChange.calledWith([null, null])).to.be.true;
+        });
+    });
+
+    describe("when controlled", () => {
+        it("Ignores defaultValue if it is also set", () => {
+            const { root } = wrap(<DateRangeInput
+                defaultValue={DATE_RANGE_2}
+                value={DATE_RANGE}
+            />);
+            assertInputTextsEqual(root, START_STR, END_STR);
+        });
+
+        it("shows empty fields when value is [null, null]", () => {
```

Comment:
caps or no caps to start test description? i think this is changing if the `it` is considered part of the sentence but looks off